### PR TITLE
Chore: Update link for Microsoft's company URL. Fixes #837

### LINKED ISSUE #837: Microsoft's company url redirects to another Microsoft url
Ran the spec through the [w3 validator](https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2Fappmanifest%2F&hide_type=all&recursive=on&depth=&check=Check) and noticed the issue below

> warning Line: 1052 https://microsoft.com/ redirected to https://www.microsoft.com/en-us/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

Sending a PR to fix this.

cc @marcoscaceres

### PR DESCRIPTION
**Pull request by janiceshiu**
Closes #837

This change (choose one):

* [ ] Breaks existing normative behavior (please add label "breaking")
* [ ] Adds new normative requirements
* [ ] Adds new normative recommendations or optional items
* [ ] Makes only editorial changes (only changes informative sections, or
  changes normative sections without changing behavior)
* [x] Is a "chore" (metadata, formatting, fixing warnings, etc).

Implementation commitment (delete if not making normative changes):

* [ ] Safari (link to issue)
* [ ] Chrome (link to issue)
* [ ] Firefox (link to issue)
* [ ] Edge (public signal)

Commit message: Chore: Update link for Microsoft's company URL. Fixes #837

(Fill in. If making normative changes, describe exactly what the behavioral
difference will be.)

cc @marcoscaceres

***
<a href="https://pr-preview.s3.amazonaws.com/janiceshiu/manifest/pull/838.html" title="Last updated on Jan 6, 2020, 8:44 AM UTC (d2e20c9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/838/83fd72b...janiceshiu:d2e20c9.html" title="Last updated on Jan 6, 2020, 8:44 AM UTC (d2e20c9)">Diff</a>

### CODE REVIEW

**Comment by kenchris** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html [hunk 1]
```diff
@@ -56,7 +56,7 @@
         {
           name: "Aaron Gustafson",
           company: "Microsoft Corporation",
-          companyURL: "https://microsoft.com/",
+          companyURL: "https://www.microsoft.com/en-us/",
```
Comment (kenchris) [inline index.html]: Is this the url we should use @aarongustafson ? or is there a better one?

Comment (janiceshiu) [reply index.html]: There is also another Microsoft contributor that uses the same URL. If there is a better URL to use, should I also update this person's Microsoft URL? Thanks.

https://github.com/w3c/manifest/blob/d2e20c9ab0983ec9d17c2434c19e27ea0f9359b6/index.html#L18-L24

Comment (kenchris) [reply index.html]: OK let's go with that for now then

Comment (aarongustafson) [reply index.html]: I would suggest the root domain as not everyone who follows it will have en-us as their primary language.

**Pull request closed without merging**